client/src/components/_common/Button/Button.jsx

```diff
@@ -24,6 +35,7 @@ const Button = ({
   iconNameAfter,
   type,
   size,
+  dataTestid,
```

Comment:
**Extra:** So that other elements can identify their `<Button >` instance in test cases.